docker-compose.yml

```diff
@@ -44,6 +44,8 @@ services:
     image: rediscommander/redis-commander:latest
     # build: .
     restart: always
+    depends_on:
+      - redis
```

Comment:
Since you are touching this file, can you please also alphabetize this YAML (the sections, as well as the options within each section)?  Reviewing is always much easier when things are alphabetical, plus it helps keep us sane.

Reply:
The alphabetization looks good to me now, thanks!